src/cli/commands/test/index.ts

```diff
@@ -298,9 +309,8 @@ export default async function test(
 
     if (isIacShareResultsOptions(options)) {
       response +=
-        chalk.bold.white(
-          `Your test results are available at: ${config.ROOT}/org/${resultOptions[0].org}/projects under the name ${resultOptions[0].projectName}`,
-        ) + EOL;
+        chalk.bold.white(shareResultsOutput(iacOutputMeta as IacOutputMeta)) +
```

Comment:
The type discrepancy seems to be due to the potential `undefined` value for `iacOutputMeta`. using the `as IacOutputMeta` could also suggest that the passed type is a subset of the properties in `IacOutputMeta`, which is not the case here.
Perhaps we could resolve it in a way more suitable for `undefined`?
Something like:
```suggestion
if (iacOutputMeta && isIacShareResultsOptions(options)) {
      response +=
        chalk.bold.white(shareResultsOutput(iacOutputMeta)) +
        EOL;
    }
```
Or alternatively:
```suggestion
        chalk.bold.white(shareResultsOutput(iacOutputMeta!)) +
```

Reply:
Done, took your 2nd suggestion.